### DISCUSSION TITLE
发送文件到手机，已完成但发生错误，但手机可以发到电脑上

### DISCUSSION THREAD

**Discussion by Myacc0851** [localsend/localsend#730] [L-简体中文]
选择文件发送到手机，全部手机都是已完成，但发生错误，点了故障排除但没有用处。前几分钟还是一直好好的，想问问是怎么回事？？

**Comment by MiKoto-Railgun** [comment]
对，我也是，手机到电脑正常，但电脑到手机就报错[500] Could not save file

**Reply by 2251750422** [reply]
手机LocalSend设置→接受界面：保存目录，设置为默认下载目录就好了

**Comment by MiKoto-Railgun** [comment]
已解决
在发送页点击文件夹分享，再在弹出的开启"授予管理所有文件的权限"选项确认就行
之后就能正常接受文件了

**Reply by Myacc0851** [reply]
不是这个原因，我的是因为源文件名字太长了，把名字改短就行了

**Reply by tbthek** [reply]
> 不是这个原因，我的是因为源文件名字太长了，把名字改短就行了

我今天也遇到这个问题，手机到电脑正常，但电脑到手机就报错[500] Could not save file，但是我看了一下手机的提示信息：file exists，我这个[500]问题就是从传第二遍开始的，但是我这个保存目录中的文件，也就是第一遍传的文件已经删除了，他还是检测有问题，至于你说的源文件名字太长，修改就可以，应该是修改文件名字，local send检测这个文件不存在，或者可能就是源文件名字太长的问题，具体还得看一些手机提示信息。

**Reply by zishenFox** [reply]
> 已解决 在发送页点击文件夹分享，再在弹出的开启"授予管理所有文件的权限"选项确认就行 之后就能正常接受文件了
遇到了同样的问题，用同样的方式解决了，感谢~

**Comment by MiKoto-Railgun** [comment]
>
![Screenshot_20231015_210914](https://github.com/localsend/localsend/assets/142686220/9a9c0b35-27c3-40bf-88f3-04f0b68048ac)

**Comment by tradeEvil** [comment]
源文件名字太长了，把名字改短就行了

**Comment by shdsj5784** [comment]
我的名字很短也提示一样的错误，昨天还可以今天突然不行了

**Reply by jinkess** [reply]
在默认存储位置就可以，也可以在默认存储文件夹下创建新文件夹也可以

**Comment by jinkess** [comment]
在默认存储位置就可以，也可以在默认存储文件夹下创建新文件夹也可以，只要在默认文件夹里随便就行

**Comment by Ginfai** [comment]
把WiFi设置中的MAC地址修改为设备MAC

**Comment by ewoks** [comment]
English please everybody, that much you all can
